README.md

```diff
@@ -1,70 +1,43 @@
-# Getting Started with Create React App
+# Math magicians
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This project is a Javascript web App that simulates math calculators, built using React js framework .
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
 
-### `npm start`
+-React js
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Getting Started
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+**To get a local copy up and running follow these simple example steps.**
 
-### `npm test`
+Clone the repo with `git clone git@github.com:souad988/math-magician.git`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Run `npm install` from the command line
```

Comment:
- [ ]  It will be nice to add explicit steps on how to clone and run your app locally as this will be of help to none technical users. For example: 
- `git clone https://github.com/souad988/math-magician.git`
- `cd math-magician`
- `git checkout calculator`
- `npm install`
- `npm start` ......